pkg/crc/segment/segment.go

```diff
@@ -34,6 +35,13 @@ func newCustomClient(config *crcConfig.Config, telemetryFilePath, segmentEndpoin
 	client, err := analytics.NewWithConfig(WriteKey, analytics.Config{
 		Endpoint: segmentEndpoint,
 		Logger:   &loggingAdapter{},
+		DefaultContext: &analytics.Context{
+			IP: net.IPv4(0, 0, 0, 0),
+			Library: analytics.LibraryInfo{
```

Comment:
You don't need to set the library. It is done automatically. See https://github.com/segmentio/analytics-go/blob/v3.0/config.go#L162

Reply:
Ah yes, I did check that code but I thought it override whatever in DefaultContext but it just add the `Library` part.